lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -102,6 +102,14 @@ def get_upload_token(app_id, binary_path)
         return upload_token_format(response.body[:appId], response.body[:projectNumber], binary_hash)
       end
 
+      # Uploads the app binary to the Firebase API
+      #
+      # args
+      #   app_id - Firebase App ID
+      #   binary_path - Absolute path to your app's apk/ipa file
+      #   platform - android vs ios
```

Comment:
how about `'android' or 'ios'` to make it clear that one of two string values is expected